python/tchannel/tornado/dispatch.py

```diff
@@ -54,10 +52,10 @@ def __init__(self):
         self.endpoints = {}
 
     @tornado.gen.coroutine
-    def _call_endpoint(self, endpoint, request, response):
+    def _call_endpoint(self, endpoint, request, response, proxy):
         try:
             yield tornado.gen.maybe_future(
-                endpoint.handler(request, response, endpoint.opts))
+                endpoint.handler(request, response, proxy))
         except:
             # TODO send application error
             pass
```

Comment:
all I change is to make the argument consistent. Actually this piece of code is not executed and override by the subclass. I plan to delete in next small patch. But I really don't want to do anything beyond the purpose of this patch. Don't be stuck at code review by adding more works